app/inc/constants.php

```diff
@@ -17,6 +17,7 @@
 define('CACHE_ENABLED', isset($_GET['nocache']) ? false : true);
 define('CACHE_PATH',    INSTALL_ROOT . 'cache/');
 define('APP_SCHEME',    isset($_SERVER['HTTPS']) && $_SERVER['HTTPS'] != 'off' ? 'https://' : 'http://');
+define('API_ROOT',   'http://' . htmlspecialchars($_SERVER['HTTP_HOST'], ENT_QUOTES, 'UTF-8') . '/api/v1/');
```

Comment:
We could reuse the APP_SCHEME defined above instead of hardcoding http.
Also, I think this won’t work with setups like http://server.com/some/path/transvision/api/v1. Do we really need the host name here?
+ nit: indent